static/styles/rendered_markdown.scss

```diff
@@ -444,6 +444,30 @@
         padding: 5px;
         border: none;
     }
+
+    /* TODO: Consolidate with .copy_message */
+    /* Style copy-to-clipboard button inside codeblocks */
+    .copy_codeblock,
+    .copy_codeblock:focus {
```

Comment:
Do we need the `:focus` line here?  I would think it doesn't do anything to specify the same style here.

Reply:
Yup, you are right. This is also there in the `.copy_message` styling. I'll remove it when we unify these styles.